README.md

```diff
@@ -1,137 +1,56 @@
 # Jaeger OpenShift Templates
 
 ## Development setup
-
 This template uses an in-memory storage with a limited functionality for local testing and development.
 Do not use this template in production environments.
 
-To directly install everything you need:
+Install everything in the current namespace:
 ```bash
-oc new-project jaeger-infra
 oc process -f https://raw.githubusercontent.com/jaegertracing/jaeger-openshift/master/all-in-one/jaeger-all-in-one-template.yml | oc create -f -
 ```
 
-Once everything is ready, the Jaeger Query UI can be accessed via http://jaeger-jaeger-infra.127.0.0.1.nip.io/ .
+Once everything is ready, `oc status` tells you where to find Jaeger URL.
```

Comment:
hmm.. I don't want to, they might change something, we might change something and it will get outdated. We do not define a lot of routes so when user will see jaeger it will be the thing he is looking for :).